libp2p/stream/chronosstream.nim

```diff
@@ -65,13 +71,29 @@ template withExceptions(body: untyped) =
     # TODO https://github.com/status-im/nim-chronos/pull/99
     raise newLPStreamEOFError()
 
+proc trackPeerIdentity(s: ChronosStream) =
+  if not s.tracked:
+    if not isNil(s.peerInfo) and s.peerInfo.agentVersion.len > 0:
+      # / seems a weak "standard" so for now it's reliable
+      s.shortAgent = s.peerInfo.agentVersion.split("/")[0]
```

Comment:
we need to limit this to a small (5) number of agents, and "others", else remotes can create metric labels

Reply:
Yeah, lets just add a list of known agent names and go with that. This list https://github.com/status-im/nim-libp2p/pull/458#issuecomment-734601883, should have most/all the current agent strings, so let's just take whatever comes before the fist `/` as known agent names and go with that.